code/modules/religion/rites.dm

```diff
@@ -72,16 +74,16 @@
 	if(!ismovable(religious_tool))
 		to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")
 		return FALSE
-	var/atom/movable/religious_tool2
+	var/atom/movable/religious_tool2 = religious_tool
 	if(!religious_tool2?.buckled_mobs?.len)
```

Comment:
probably wanna check can_buckle beforethis as well and send this again to_chat(user, "<span class='warning'>This rite requires a religious device that individuals can be buckled to.</span>")